### ISSUE TITLE
The New Reflection - blog series

### ISSUE THREAD

**Issue by dmlloyd** [dmlloyd/dmlloyd.github.io#4]
Begin a series on the new `MethodHandle`-based reflection, based on the talk I gave.

- [x] #9
- [x] #10
- [x] #11

**Comment by SirYwell** [comment]
Hi, I just went through part 1 and noticed that some code snippets, e.g. https://github.com/dmlloyd/dmlloyd.github.io/blob/cfeeb5a575d722192a8e496fc5e5db968fb786fb/content/posts/2024-11-08-the-new-reflection.adoc?plain=1#L77 won't compile - `Object` cannot be converted to `int`. Given the unused return type, you could probably also mention `MethodHandles.dropReturn` (if you didn't already plan to bring that up later).

**Comment by dmlloyd** [comment]
Good catch, thanks. I guess I rushed it too much. The fix is being pushed up now.

I intend to mention `dropReturn` and other transformations in the next post.
